<?xml version="1.0"?>

<launch>
	<node name="cancel_hovering" pkg="vision" type="cancel_hovering" output="screen">
        <param name="cmd_topic_vel" value="/cmd_vel"/>
        <param name="ardrone_navdata" value="/ardrone/navdata"/>
    </node>

     <!-- Start rviz visualization with preset config -->
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find vision)/rviz_config/realNoPred.rviz -f centre"/> -->
</launch>